docs/multiple_dbs.rst

```diff
@@ -39,3 +39,37 @@ an issue where you want to track the history on a table that lives in a separate
 database to your user model. Since Django does not support cross-database relations,
 you will have to manually track the ``history_user`` using an explicit ID. The full
 documentation on this feature is in :ref:`Manually Track User Model`.
+
+Migrating Historical Tables to Multiple Databases
+-------------------------------------------------
+If you want your historical records to live in separate databases, you will need
+to write a [database router](https://docs.djangoproject.com/en/2.1/topics/db/multi-db/#database-routers).
+The router interface requires 4 methods, but the only one that concerns us here
```

Comment:
This isn't right. You can probably just remove this entire section on routers and only include the part on the parameter below

Reply:
@dopatraman

Reply:
Technically, one would still need a database router to forward the historical table migrations right?

Reply:
Yep, but they'd need all 4 methods. Or at least 3 of them. If they only migrate to the other table and then can't write or read to it, there'll be issues..

Reply:
But that's pretty much basic Django functionality. So don't really think we need a section on it

Reply:
Done.